src/server/scripts/Northrend/Ulduar/Ulduar/boss_kologarn.cpp

```diff
@@ -624,6 +613,17 @@ class boss_kologarn_eyebeam : public CreatureScript
 
         void UpdateAI(uint32 diff)
         {
+            if (justSpawned)
+            {
+                me->DespawnOrUnsummon(10000);
+
+                if (Creature* cr = ObjectAccessor::GetCreature(*me, m_pInstance->GetData64(TYPE_KOLOGARN)))
+                    me->CastSpell(cr, me->GetEntry() == NPC_EYE_LEFT ? SPELL_FOCUSED_EYEBEAM_LEFT : SPELL_FOCUSED_EYEBEAM_RIGHT, true);
+                me->CastSpell(me, SPELL_FOCUSED_EYEBEAM, true);
+
+                me->GetMotionMaster()->MoveChase(me->GetMap()->GetPlayer(eyebeamTarget->GetGUID()));
+                justSpawned = false;
+            }
```

Comment:
```cpp
            if (justSpawned)
            {
                me->DespawnOrUnsummon(10000);
                if (Creature* cr = ObjectAccessor::GetCreature(*me, m_pInstance->GetData64(TYPE_KOLOGARN)))
                {
                    me->CastSpell(cr, me->GetEntry() == NPC_EYE_LEFT ? SPELL_FOCUSED_EYEBEAM_LEFT : SPELL_FOCUSED_EYEBEAM_RIGHT, true);
                }
                me->CastSpell(me, SPELL_FOCUSED_EYEBEAM, true);
                me->GetMotionMaster()->MoveChase(me->GetMap()->GetPlayer(eyebeamTarget->GetGUID()));
                justSpawned = false;
            }
```

We are now using them even for a single line since some months ago from now on, let's save spaces.

Reply:
@wizzymore  Please see above, still needed.

Reply:
Hi @Si1ker,
Thanks for the feedback, but, can you please highlight is missing?
I updated the script and removed the blank lines that were not needed

Reply:
Please see this commit right here:
https://github.com/azerothcore/azerothcore-wotlk/pull/3529/commits/33a151675a4cbbd39d3dd49921bf9b0863c926bd

Reply:
@wizzymore 

instead of...

```cpp
                if (Creature* cr = ObjectAccessor::GetCreature(*me, m_pInstance->GetData64(TYPE_KOLOGARN)))
                    me->CastSpell(cr, me->GetEntry() == NPC_EYE_LEFT ? SPELL_FOCUSED_EYEBEAM_LEFT : SPELL_FOCUSED_EYEBEAM_RIGHT, true);
```

use...

```cpp
                if (Creature* cr = ObjectAccessor::GetCreature(*me, m_pInstance->GetData64(TYPE_KOLOGARN)))
                {
                    me->CastSpell(cr, me->GetEntry() == NPC_EYE_LEFT ? SPELL_FOCUSED_EYEBEAM_LEFT : SPELL_FOCUSED_EYEBEAM_RIGHT, true);
                }
```


I said before in the first changed requested, "We are now using them even for a single line since some months ago from now on" , referring to brackets { } 
We are adapting this as a new standard.

Reply:
Ohh, sure, will change this now